src/NuGet.Clients/VisualStudio/Extensibility/IVsPackageInstallerProjectEvents.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace NuGet.VisualStudio
+{
+    /// <summary>
+    /// Contains batch events which are raised when packages are installed or uninstalled from projects and the current
```

Comment:
The summary should mention packages.config, and so should the events. I see it is updated for IVsPackageProjectMetadata, but the caller would only discover that after using the IVsPackageInstallerProjectEvents api